src/main/kotlin/com/team4099/robot2022/commands/led/ChangeLedState.kt

```diff
@@ -0,0 +1,56 @@
+package com.team4099.robot2022.commands.led
+
+import com.team4099.robot2022.config.Constants
+import com.team4099.robot2022.subsystems.LED
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class ChangeLedState(val changestate: Constants.LED.LEDState) : CommandBase() {
+  init {
+    addRequirement(Intake)
+    addRequirement(Shooter)
```

Comment:
remove as we are only going to be refernecing their values and not call any methods from it